assignments/objects.js

```diff
@@ -19,46 +19,140 @@ let example = {
 
 // Write your intern objects here:
 
+let mitzy = {
+  id: 1,
+  name: "Mitzy",
+  email: "mmelloy0@psu.edu",
+  gender: "F"
+}
+let kennan = {
+  id: 2,
+  name: "Kennan",
+  email: "kdiben1@tinypic.com",
+  gender: "M",
+  speak: function speak(){
+    phrase = "Hello my name is" + " " + this.name;
+    return phrase;
+}}
+let keven = {
+  id: 3,
+  name: "Keven",
+  email: "kmummery2@wikimedia.org",
+  gender: "M",
+}
+let gannie = {
+  id: 4,
+  name: "Gannie",
+  email: "gmartinson3@illinois.edu",
+  gender: "M"
+}
+let antonietta = {
+  id: 5,
+  name: "Antonietta",
+  email: "adaine5@samsung.com",
+  gender: "F",
+  multiplyNums: function multiplyNums(x, y){
+    return x * y;
+  }
+}
+
+
 
 // ==== Challenge 2: Reading Object Data ==== 
 // Once your objects are created, log out the following requests from HR into the console:
 
 // Mitzi's name
 
+console.log(mitzy.name);
+
 // Kennan's ID
 
+console.log(keven.id);
+
 // Keven's email
 
+console.log(keven.email);
+
 // Gannie's name
 
+console.log(gannie.name);
+
 // Antonietta's Gender
 
+console.log(antonietta.gender);
+
 // ==== Challenge 3: Object Methods ==== 
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
-// console.log(kennan.speak());
+
+
+console.log(kennan.speak());
+
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
 
-// === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
+console.log(antonietta.multiplyNums(3,4));
 
+// === Great work! === Head over to the the arrays.js file or take a look at the stretch challenge
 // ==== Stretch Challenge: Nested Objects and the this keyword ==== 
 
 // 1. Create a parent object with properties for name and age.  Make the name Susan and the age 70.
 // 2. Nest a child object in the parent object with name and age as well.  The name will be George and the age will be 50.
 // 3. Nest a grandchild object in the child object with properties for name and age.  The name will be Sam and the age will be 30
 // 4. Give each of the objects the ability to speak their names using the this keyword.
 
-let parent = {}
+let child = {};
+let grandchild = {};
+
```

Comment:
So here you could have done something like:

function speakFunc() {
    return 'I'm ${this.name}';
}

And then in each object, like:

let parent = {
    name: "Susan",
    age: 70,
    speak: speakFunc,
}

Notice that I didn't do `speak: speakFunc()` because that would have called the function and stored the result of `speakFunc()` into `speak` which has a totally different effect. This is what Trevor and I were talking about last night during standup for some time.